### ISSUE TITLE
[Adaptive Card iOS - ColumnSet.Input.Text.InlineAction.VerticalStretch]: Voice over does remain silent and does not announce about the comment edit field.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#7019]
https://microsoft.visualstudio.com/OS/_workitems/edit/37841191

**Comment by jwoo-msft** [comment]
duplicate of #7024

**Comment by vagpt** [comment]
Closing this issue due to duplicate of #7024
